index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <!-- Header Section Start -->
+  <header id="main">
+    <h1 id="title">Dr. Norman Borlaug</h1>
+    <div id="img-div">
+      <a id="tribute-link" target="_blank" rel="noopener noreferrer"
+        href="https://www.google.com/search?q=images&rlz=1C1BNSD_enPK978PK978&sxsrf=APq-WBsTt8cnGYJwODkbkBjILQtY6Wj-Mw:1650441386626&tbm=isch&source=iu&ictx=1&vet=1&fir=2nDXavJs9DoKTM%252CB51x0PBR9KNzvM%252C_%253BsPwUW2x5Z3mupM%252CnBiD9BWYMB87aM%252C_%253BITUG1uFAV1aQ9M%252CX9GBbDAacv-dWM%252C_%253ByRdsZ-zqxwGPmM%252CMP_pVERxQybf_M%252C_%253Bn5hAWsQ-sgKo_M%252C-UStXW0dQEx4SM%252C_%253BISsdmwh92GPtrM%252CtnVTsEa64LdCyM%252C_%253BIizY1BpQ3iE_iM%252C3jhko1yxn_N42M%252C_%253BUVAHTXdge9JbrM%252CtnVTsEa64LdCyM%252C_%253B0sOgRvZZyWRMuM%252CHQreBDBQV72l4M%252C_%253BarFfSjMu_GX7sM%252CzB6OZ3HGfiBMnM%252C_%253Bez-ubljHwN9MSM%252Cm4H9nlxeVf5uvM%252C_%253Bi0x7moJaCtF5XM%252Cfb09TtLq3UJOWM%252C_%253BEd4EHROkh7SA0M%252Cm4H9nlxeVf5uvM%252C_%253BpZG009_n8yw8iM%252CHgiYJMPP6P-sHM%252C_&usg=AI4_-kTxIZQbgQRPuw5heBv9k6CvxltYPA&sa=X&ved=2ahUKEwj-88mklaL3AhUGV8AKHS1DCBcQ9QF6BAgYEAE#imgrc=IizY1BpQ3iE_iM">
+        <img id="image" alt="header-images"
+          src="https://images.pexels.com/photos/302743/pexels-photo-302743.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500">
+      </a>
+    </div>
+    <p id="img-caption">This project is for microverse coding challenges</p>
+  </header>
```

Comment:
Header Content is changed in the layout ♻️. used header tag for the separation of the heading area in the layout and use h1 tag 🏷️  for the heading text. and also change the place of img tag inside a div for more styling of the image.

Reply:
- [ ] [optional] @samiullah997 I can't clearly understand the comment above. kindly edit it or remove it since you have another comment for this section.